cyences_app_for_splunk/default/props.conf

```diff
@@ -553,7 +553,7 @@ EVAL-dest_name = mvappend(DNS, NETBIOS)
 # For Cyences Assets datamodel
 FIELDALIAS-asset_id = HOST_ID AS asset_id
 FIELDALIAS-asset_ip = IP AS asset_ip
-
+LOOKUP-qualys_kb_lookup = qualys_kb_lookup QID OUTPUT TITLE as description
```

Comment:
@VatsalJagani @hardikhdholariya The issue here is that if customer is not using qualys then it will generate lookup error. I faced same issue earlier for qualys iteself.

Can you make sure description field is not extracted by other. that might cause issue for CIM.

Reply:
Deleted.